### PR TITLE
Course Consolidation

### PR DESCRIPTION
**Pull request by BenWoodford76**
Consolidate courses with the degree level at the start to enable ease of removal for enhanced course search.

as per #643

### CODE REVIEW

File: _people/daniel_morris.md [hunk 1]
```diff
@@ -4,5 +4,5 @@ headshot: cxWKwjn
 gender: male
 submitted: false
 course:
-  - Physics MSc
+  - MSc Physics
```
Comment (samozzy) [inline _people/daniel_morris.md:7]: What’s the difference between an MSc and an MSci?

File: _people/felicity_chilver.md [hunk 2]
```diff
@@ -6,5 +6,5 @@ award: Fellowship
 submitted: false
 graduated: 2018
 course:
-  - English Literature MA
+  - MA English Literature
```
Comment (samozzy) [inline _people/felicity_chilver.md:9]: I assume there’s also a BA English that should be here?

File: _people/george_waring.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ graduated: 2018
 award: Merit
 gender: male
 course:
-  - International Relations BA Hons
+  - BA Hons International Relations
```
Comment (samozzy) [inline _people/george_waring.md:8]: Is a BA and a BA Hons the same?

File: _people/harry_robertson.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: Harry Robertson
 headshot: kC5wJPX
 gender: male
 course: 
-  - English and Classical Civilisation BA (Hons)
-  - Film, Screen and Television Industries MA
+  - BA Hons English and Classical Civilisation
```
Comment (samozzy) [inline _people/harry_robertson.md:6]: BA vs BA Hons?

File: _people/james_pheasey.md [hunk 5]
```diff
@@ -4,9 +4,11 @@ gender: male
...
-  - Philosophy BA
-  - Philosophy MA
+  - BA Philosophy
+  - MA Philosophy
 contact_allowed: true
+
+# SU President (2019/2020)
 ---
 
-SU President (2019/2020)
```
Comment (samozzy) [inline _people/james_pheasey.md:12]: Why comment this?

File: _people/jennifer_danaher.md [hunk 6]
```diff
@@ -2,5 +2,5 @@
 title: Jennifer Danaher
 gender: female
 course:
-  - Psychology BSc (Hons)
+  - BSc Hons Psychology
```
Comment (samozzy) [inline _people/jennifer_danaher.md:5]: Hons etc

File: _people/katie_rowley-jones.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 title: Katie Rowley-Jones
 gender: female
 course:
-  - Law BA (Hons)
+  - BA Hons Law
```
Comment (samozzy) [inline _people/katie_rowley-jones.md:5]: Hons etc

File: _people/nick_gill.md [hunk 8]
```diff
@@ -7,5 +7,5 @@ award: Fellowship
 submitted: 04/01/2017
 contact_allowed: true
 course:
-  - Politics (Hons) BA
+  - BA (Hons) Politics
```
Comment (samozzy) [inline _people/nick_gill.md:10]: Hons?

File: _people/daniel_morris.md (hunk 1 above)
Comment (samozzy) [reply _people/daniel_morris.md:7]: Specifically given Dan and David graduated close to one another and whether UoN offers a MSc Physics

File: _people/felicity_chilver.md (hunk 2 above)
Comment (BenWoodford76) [reply _people/felicity_chilver.md:9]: Not necessarily? I Thought if you did a straight 4 year masters then you skip the BA bit

File: _people/george_waring.md (hunk 3 above)
Comment (BenWoodford76) [reply _people/george_waring.md:8]: I think so, but it we're stripping the degree level for the search function anyway then it's not really an issue.

think you can get a BA Hons in a straight subject but only a BA if you do joint Honours

File: _people/james_pheasey.md (hunk 5 above)
Comment (BenWoodford76) [reply _people/james_pheasey.md:12]: if it was part of a larger Bio then i'd leave it but it seems a bit odd just being there by itself, have no strong feelings so am happy to revert

File: _people/daniel_morris.md (hunk 1 above)
Comment (BenWoodford76) [reply _people/daniel_morris.md:7]: Seems like you do a BSc but an MSci

File: _people/harry_robertson.md (hunk 4 above)
Comment (BenWoodford76) [reply _people/harry_robertson.md:6]: see above

File: _people/felicity_chilver.md (hunk 2 above)
Comment (samozzy) [reply _people/felicity_chilver.md:9]: I guess there’s lots of nuance: sciences definitely have an integrated option, though Jess D did it separately (whereas David/Dan did not); and English is often optional (Callum has 2 degrees), but I suppose could be straight through.

File: _people/james_pheasey.md (hunk 5 above)
Comment (samozzy) [reply _people/james_pheasey.md:12]: Can it go in careers?

File: _people/george_waring.md (hunk 3 above)
Comment (samozzy) [reply _people/george_waring.md:8]: Okay - can we therefore have all the Hons in brackets? (Stripping them out with brackets feels more technically foolproof than without)

File: _people/felicity_chilver.md (hunk 2 above)
Comment (BenWoodford76) [reply _people/felicity_chilver.md:9]: my thought iswe leave it as is unless she submits a bio which alters it?

**Review by samozzy — APPROVED** [review]

**Pull request merged**